src/fft.jl

```diff
@@ -176,6 +176,10 @@ end
     ccall((:fftwf_plan_with_nthreads,libfftw3f), Cvoid, (Int32,), nthreads)
 end
 
+function get_num_threads()
+    ccall((:fftw_planner_nthreads,libfftw3), Cint, ())
```

Comment:
shouldn't this have an `fftw_vendor` check?

Reply:
Yeah it should